server/noncommercial-schema.es6

```diff
@@ -0,0 +1,330 @@
+var noncommercialSchema = {
+  'id':'/noncommercialPermit',
+  'type':'object',
+  'allOf':[
+    { '$ref': '/commonFields'},
+    {
+      'properties':{
+        'applicantInfo': {
+          '$ref': '/noncommercialApplicantInfo'
+        },
+        'type': {
+          'default': 'noncommercial',
+          'enum':[
+            'noncommercial',
+            'tempOutfitters'
+          ],
+          'type': 'string'
+        },
+        'noncommercialFields': {
+          '$ref': '/noncommercialFields'
+        }
+      },
+      'required': ['applicantInfo', 'type', 'noncommercialFields']
+    }
+  ]
+};
+
+var noncommercialApplicantInfoSchema = {
+  'id': '/noncommercialApplicantInfo',
+  'type': 'object',
+  'allOf':[
+    {'$ref':'/applicantInfoBase'},
+    {
+      'properties': {
+        'organizationName': {
+          'default':'',
+          'type': 'string'
+        },
+        'orgType':{
+          'default':'',
+          'description':'Organization Type',
+          'enum':[
```

Comment:
I think this enum doesn't match the options in the middle layer API right now. Could you check @sethalt?